flexmeasures/data/services/scheduling.py

```diff
@@ -312,6 +312,9 @@ def create_sequential_scheduling_job(
         connection=current_app.queues["scheduling"].connection,
     )
 
+    # Stand-in for MultiStorageScheduler
```

Comment:
Nice! 

I wonder if we should reflect that the output of the schedules used a sequential approach..